.pubignore

```diff
@@ -0,0 +1,4 @@
+.vscode
```

Comment:
Does the file in the root folder works for every package? the publish CLi is called within a folder, so not sure this would be respected.
Maybe we'd need a single file with symlinks

Reply:
Yes, I've tested with `dart pub publish --dry-run` to verify the file is picked up